### PR TITLE
Add deprecation message for skip_delete for Project resource

### PR DESCRIPTION
**Pull request by abd-goog**
Add deprecation message for skip_delete for Project resource. 
Part of b/330143705

Part of  https://github.com/hashicorp/terraform-provider-google/issues/18775

**Release Note Template for Downstream PRs (will be copied)**

```release-note:deprecation
'skip_delete' is being deprecated in favor of a new field 'deletion_policy' in the next major release
```

### CODE REVIEW

File: mmv1/third_party/terraform/services/resourcemanager/resource_google_project.go [hunk 1]
```diff
@@ -68,6 +68,7 @@ func ResourceGoogleProject() *schema.Resource {
 			},
 			"skip_delete": {
 				Type:        schema.TypeBool,
+				Deprecated:  `'skip_delete' is deprecated and will be removed in a future major release. The new release adds support for 'deletion_policy' instead.`,
```
Comment (slevenick) [inline mmv1/third_party/terraform/services/resourcemanager/resource_google_project.go:71]: I think you can remove quotation marks in this message and it should fix the compilation error

Comment (abd-goog) [reply mmv1/third_party/terraform/services/resourcemanager/resource_google_project.go]: Thanks, removed quotation marks. Doesn't seem to help though.

Comment (abd-goog) [reply mmv1/third_party/terraform/services/resourcemanager/resource_google_project.go]: Locally the provider generation works fine. There's something wrong with the test, it's picking up older version perhaps.

**Comment by slevenick** [comment]
@melinath do you know why TGC would be failing for this change? It doesn't seem like it should be causing a failure

**Comment by melinath** [comment]
This is caused by https://github.com/hashicorp/terraform-provider-google/issues/17924 - rerunning the tests should resolve it. (https://github.com/GoogleCloudPlatform/magic-modules/pull/10671 will fix it more permanently.) /gcbrun

**Review by slevenick — APPROVED** [review]

**Pull request merged**